dbms/src/Storages/KVStore/MultiRaft/Persistence.cpp

```
@@ -215,19 +220,33 @@ bool KVStore::canFlushRegionDataImpl(
     if (can_flush && flush_if_possible)
     {
         // This rarely happens when there are too may raft logs, which don't trigger a proactive flush.
+<<<<<<< HEAD
         LOG_INFO(
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If LOG_INFO is a macro then please configure it.

(unknownMacro)